### PR TITLE
Add method to generate point between circles

### LINKED ISSUE #9: Method to generate a point between circles
it will be awesome to have an api that return a point between 2 circle - inner and outer circle!

### PR DESCRIPTION
**Pull request by khubo**
This issues should close #9. Somehow I cannot get the test coverage to get to 100%. advice?

### CODE REVIEW

**Comment by sagi** [comment]
Thanks @khubo!

File: src/index.js [hunk 1]
```diff
@@ -69,6 +69,28 @@ const randomCirclePoint = (centerPoint, radius, randomFn = Math.random) => {
   );
 };
 
+/**
+ *
+ * Given a center point C and a smaller radius and bigger radius, return a point that is
+ *  between the two circles
+ */
+const randomPointBetweenCircles = (
```
Comment (sagi) [inline src/index.js]: Can you please change `randomPointBetweenCircles` to `randomAnnulusPoint`? [Annulus](https://en.wikipedia.org/wiki/Annulus_(mathematics)) seems like the correct math term for this.

In the comment above could please add something like:

```js
/**
* Returns a random point in a region bounded by two concentric circles (annulus). 
*  ...
*/
```

File: src/index.js [hunk 2]
```diff
@@ -69,6 +69,28 @@ const randomCirclePoint = (centerPoint, radius, randomFn = Math.random) => {
...
+  centerPoint,
+  innerRadius,
+  outRadius,
```
Comment (sagi) [inline src/index.js]: Just for consistency, can you please change `outRadius` to `outerRadius` ?

File: src/index.js [hunk 3]
```diff
@@ -69,6 +69,28 @@ const randomCirclePoint = (centerPoint, radius, randomFn = Math.random) => {
...
+  randomFn = Math.random
+) => {
+  if (innerRadius > outRadius) {
+    throw new Error('innerRadus should be smaller');
+  }
+
+  return randomCircumferencePoint(
+    centerPoint,
+    Math.floor(Math.random() * (outRadius - innerRadius + 1) + innerRadius),
```
Comment (sagi) [inline src/index.js]: While this seems correct, we need to verify that it results in points that are uniformly distributed on the annulus. I'll do the calculation later today to verify :)

**Comment by khubo** [comment]
@sagi i have fixed as per your comments! :smile_cat:

**Pull request merged**